apiary.apib

```diff
@@ -59,19 +58,91 @@ Any response that does not provide a ```200 OK``` response code should be treate
 
 ### GET
 
-+ Response 200 (application/json)
++ Response 200 (application/json; charset=utf-8)
     + Headers
-    
+
             X-Dude-Api-Limit: Unlimited or Number
             X-Dude-Api-Remaining: Unlimited or Number
             X-Dude-Api-Server: Api Server ID
-            
+
     + Attributes (Heartbeat)
 
+
+# Group Inventory Item
+
+## Inventory Item Collection [/accounts/{accountId}/inventoryitems{?page,pagesize,orderby}]
```

Comment:
Endpoints should have description. See https://readme.dudesoln.com/display/EA/Apiary+Documentation+Standards